### PR TITLE
Translations

### PR DESCRIPTION
**Pull request by hyperupcall**
_No description needed_.

### CODE REVIEW

File: src/server/services/csvPipeline/success.js [hunk 1]
```diff
@@ -12,7 +14,7 @@
  */
 function success(req, res, comment = '') {
 	res.status(200) // 200 OK
-	.send(`<h1>SUCCESS</h1>${comment}`);
+	.send(`<h1>${translate('status.success')}</h1>${comment}`);
```
Comment (mikelbrierly) [inline src/server/services/csvPipeline/success.js:17]: Do we need the `h1` tags here still? Or should we remove them?

Comment (davidNicolas-cecs) [reply src/server/services/csvPipeline/success.js:17]: True!

Comment (JesseVarGar) [reply src/server/services/csvPipeline/success.js:17]: nice

File: src/server/services/csvPipeline/uploadMeters.js [hunk 2]
```diff
@@ -58,8 +62,10 @@ async function uploadMeters(req, res, filepath, conn) {
 		const defaultGraphicUnitName = meter[24];
 		const defaultGraphicUnitId = await getUnitId(defaultGraphicUnitName, Unit.unitType.UNIT, conn);
 		if (!defaultGraphicUnitId) {
-			const msg = `For meter ${meter[0]} the default graphic unit of ${defaultGraphicUnitName} is invalid`;
-			throw new CSVPipelineError(msg, undefined, 500);
+			throw new CSVPipelineError(translate(
+				'csv.invalid-graphic-unit',
+				{ meter: meter[0], graphic_unit: defaultGraphicUnitName }
+			), undefined, 500);
```
Comment (mikelbrierly) [inline src/server/services/csvPipeline/uploadMeters.js:68]: image

File: src/server/translations/data.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+		"csv.mismatched-cumulative-variables": "(Need French) On meter {meter} in pipeline: cumulative was false but cumulative reset was true. To avoid mistakes all reading are rejected.",
+		"csv.failed-to-parse-start-datetime": "(Need French) The start date/time of {datetime} did not parse to a date/time using the normal format so a less restrictive method is being tried. This is a warning since it can lead to wrong results but often okay.",
+		"csv.failed-to-parse-end-datetime": "(Need French) The end date/time of {datetime} did not parse to a date/time using the normal format so a less restrictive method is being tried. This is a warning since it can lead to wrong results but often okay.",
+		"csv.invalid-date": "(Need French) For meter {meter}: Error parsing Reading #{num} The start ({start}) and/or end time ({end}) provided did not parse into a valid date/time so all reading are rejected.",
+		"csv.invalid-reading-number": "(Need French) For meter {meterName}: Error parsing Reading #{num} with cumulative data. The reading value provided of {reading} is not considered a number so all reading are rejected."
+	}
```
Comment (mikelbrierly) [inline src/server/translations/data.js:37]:
```suggestion
// just an example: 
	  "es": {
	  
	  }
	}
```

Comment (mikelbrierly) [reply src/server/translations/data.js:37]: Need spanish? 

- Jesse

**Pull request merged**